api/v1beta1/awscluster_types.go

```diff
@@ -56,33 +56,46 @@ type AWSClusterSpec struct {
 	// +optional
 	ControlPlaneLoadBalancer *AWSLoadBalancerSpec `json:"controlPlaneLoadBalancer,omitempty"`
 
-	// ImageLookupFormat is the AMI naming format to look up machine images when
-	// a machine does not specify an AMI. When set, this will be used for all
-	// cluster machines unless a machine specifies a different ImageLookupOrg.
-	// Supports substitutions for {{.BaseOS}} and {{.K8sVersion}} with the base
-	// OS and kubernetes version, respectively. The BaseOS will be the value in
-	// ImageLookupBaseOS or ubuntu (the default), and the kubernetes version as
-	// defined by the packages produced by kubernetes/release without v as a
-	// prefix: 1.13.0, 1.12.5-mybuild.1, or 1.17.3. For example, the default
+	// AMI ID is the first method for finding an AMI ID, if this field is set all other methods will be ignored.
+	// example: ami-1234567890abcdef0
+	AMI AMIReference `json:"ami,omitempty"`
```

Comment:
Am I seeing this incorrectly? This should be `infrav1.AMIReference`. I don't see you pulling that type in here.

Reply:
No, wait. Interesting. This didn't exist in v1beta1. But it's part of the exp package.

Reply:
So... this resource accepts filters. Which you can use to define something like Architecture = arm64. Is there a reason we can't use that? The ID and ARN are optional parameters.